docs/manual/docs/maintainer-guide/updating/index.md

```diff
@@ -1,7 +1,170 @@
+        * Application server running GeoNetwork
+        * Elasticserach
+    
+    2. When using application data directory be sure to backup  **`/geonetwork/WEB-INF/data/config/encryptor/encryptor.properties`** 
+
+=== "External data directory"
+
+    Before you start:
+    
+    1. Stop the services:
+        
+        * Application server running GeoNetwork
+        * Elasticserach
+Update:
+
+=== "Application data directory"
+    
+    1. GeoNetwork 4.4 requires a Java 11 environment (JRE) to be installed on your system.
+       
+        Later versions of Java will not work at present. This must be done prior to installation.
+       
+    2. This release of GeoNetwork 4.4 is developed and tested with Elasticsearch `7.17.15`.
+    
+        If required update Elasticserach, see [Installing search platform](../../install-guide/installing-index.md).
+    
+    3. Minor update may be performed in place, update **`geonetwork`** according to [installation approach](../../install-guide/index.md) used.
+        
+        !!! note
+        
+            Configurations changes made to application  **`/WEB-INF/`** files should be backed up, and the same configuration changes reapplied to the new application.
+        
+            With the service stopped you may find it easier to manually deploy (by unzipping the **`geonetwork.war`**
+            into the correct **`webapps/geonetwork/`** location) in order to apply configuration changes to **`/WEB-INF/**`.
+       
+            Read more at [Configuration](../../install-guide/configuring-database.md).
+
+    4. Restore the backup of **`encryptor.properties`** to the application data directory.
+
+    5. Restart services:
+        
+        * Elasticsearch
+        * Application server for GeoNetwork
+
+=== "External data directory"
+
+    1. GeoNetwork 4.4 requires a Java 11 environment (JRE) to be installed on your system.
+       
+        Later versions of Java will not work at present. This must be done prior to installation.
+
+    2. This release of GeoNetwork 4.4 is developed and tested with Elasticsearch `7.17.15`.
+    
+        If required update Elasticserach, see [Installing search platform](../../install-guide/installing-index.md).
+
+    3. Minor update may be performed in place, update **`geonetwork`** according to [installation approach](../../install-guide/index.md) used.
+       
+        !!! note
+        
+            Configurations changes made to application  **`/WEB-INF/`** files should be backed up, and the same configuration changes reapplied to the new application.
+        
+            With the service stopped you may find it easier to manually deploy (by unzipping the **`geonetwork.war`**
+            into the correct **`webapps/geonetwork/`** location) in order to apply configuration changes to **`/WEB-INF/**`.
+       
+            Read more at [Configuration](../../install-guide/configuring-database.md).
+
+    4. Restart services:
+        
+        * Elasticsearch
```

Comment:
```suggestion
```

Related to previous comment, that is not required to start/stop Elasticsearch for an update.